packages/react-styled-ui/src/Text/index.js

```diff
@@ -1,23 +1,32 @@
 import React, { forwardRef } from 'react';
-import Box from '../Box';
+import PseudoBox from '../PseudoBox';
+import useColorMode from '../useColorMode';
 import useTheme from '../useTheme';
+import getColorPalette from '@trendmicro/styled-ui-theme/build/color-palette';
```

Comment:
@austinhung81 

Change it to

```js
import getColorPalette from '@trendmicro/styled-ui-theme/color-palette';
```

Note: the `build` folder is only available under the development environment.